update instruction for scenario 1 (#1568)
MicrosoftDocs/OfficeDocs-Support: Exchange/ExchangeHybrid/user-and-shared-mailboxes/mailbox-exists-exo-onpremises.md

5. Restore any custom proxy addresses and any other Exchange attributes that were stripped when the mailbox was disabled (compare to the `Get-Mailbox` command from step 1).

> 6. Collect information about GUID of mailboxes and database:

Reviewer: ```suggestion
6. Collect information about the GUID of the mailbox and database:
```

Author: Hello @JohanFreelancer9 ,
I cannot understand this one. Step 6 is about one database (on-prem) and about two mailboxes (one is on-prem, another is cloud).
Could you please check again?
Thank you

Reviewer: @MaratMussabekov - Ah, thanks for the clarification. I have changed my suggestion accordingly (just inserted the word "the". Many thanks.

Author: Hello @JohanFreelancer9 ,
I do not see any change, which one would be correct? where should I add "The"?
Also, GUID should be plural here, as we are talking about three different GUIDs. Would be "GUIDs" correct? or it would be better to leave abbreviation without "s"?
Thank you

> 6. Collect information about GUID of mailboxes and database:

Reviewer: ```suggestion
6. Collect information about the GUIDs of the mailboxes and database:
```

Reviewer: @MaratMussabekov - Yes GUIDs would be correct. It should be "the" DUIDs and "the" mailboxes. To implement these changes, please commit my latest suggested changes. Many thanks.

>     - to get GUID of disconnected mailbox, take the value of ExchangeGUID parameter from file saved in Step 2

Reviewer: ```suggestion
    - To get the GUID of the disconnected mailbox, use the value of the ExchangeGUID parameter from the file saved in Step 2.
```

>     - to get GUID of on-premises database, take the value of Database parameter from file saved in Step 2, then run the following command:

Reviewer: ```suggestion
    - To get the GUID of the on-premises database, use the value of the Database parameter from the file saved in Step 2, then run the following command:
```

  
    ```powershell
    Get-MailboxDatabase "database identity" | fl *GUID*
    ```  

>     - to get GUID of cloud mailbox, run following command by using Exchange Online PowerShell:

Reviewer: ```suggestion
    - To get the GUID of cloud mailbox, run the following command by using Exchange Online PowerShell:
```
